src/typecode/magic2.py

```diff
@@ -79,18 +79,60 @@
 TYPECODE_LIBMAGIC_DATABASE = 'typecode.libmagic.db'
 
 
+def load_lib_failover():
+  # loader from python-magic
+  # License: MIT 
+  # Copyright: Adam Hupp
+  libmagic = None
+  # Let's try to find magic or magic1
+  dll = ctypes.util.find_library('magic') \
```

Comment:
I am not a big fan of line continuations and end-of-line comments. I reckon you copied the code from upstream, butw what about instead something along these lines:
```Python
    dll = (ctypes.util.find_library('magic')
        or ctypes.util.find_library('magic1')
        or ctypes.util.find_library('cygmagic-1')
        or ctypes.util.find_library('libmagic-1')
        or ctypes.util.find_library('msys-magic-1')
    )
```